Mathlib/SetTheory/Cardinal/SchroederBernstein.lean

```diff
@@ -39,13 +39,16 @@ section antisymm
 variable {α : Type u} {β : Type v}
 
 /-- **The Schröder-Bernstein Theorem**:
-Given injections `α → β` and `β → α`, we can get a bijection `α → β`. -/
-theorem schroeder_bernstein {f : α → β} {g : β → α} (hf : Function.Injective f)
-    (hg : Function.Injective g) : ∃ h : α → β, Bijective h := by
+Given injections `α → β` and `β → α` that satisfy a pointwise property `R`, we can get a bijection
+`α → β` that satisfies that same pointwise property. -/
+theorem schroeder_bernstein' {f : α → β} {g : β → α} (hf : Function.Injective f)
```

Comment:
I admittedly do not love this naming. `schoreder_bernstein_pointwise`?

Reply:
Sounds better... but I'm also not sure what a good name is. Ask the hive mind, i.e. on zulip?

Reply:
Maybe `schroeder_bernstein_of_rel`?  I am not too sold on `rel`, but highlighting the extra hypothesis with `of` seems appropriate.

Reply:
`of_rel` is good, I could state the hypothesis as an actual `Rel \a \b` but that would require more imports, not sure if it's worth it or not.